src/web/components/ApiKeyManagement/KeyHelper.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+
+const yearInSeconds = 31556952;
+
+export const needsRotating = (apiKey: ApiKeyDTO): boolean => {
+  const currentDate = new Date().getTime();
```

Comment:
I think we can just use `Date.now()` instead, it's a bit more concise.

Reply:
Yep looks like they output the same number but Date.now() doesn't require the new keyword